translate/api/TranslateV3Samples/TranslateV3ListGlossary.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+// This is a generated sample ("RequestPagedAll", "translate_v3_list_glossary")
+
+using CommandLine;
+// [START translate_v3_list_glossary]
+using Google.Api.Gax;
+using Google.Cloud.Translate.V3;
+using System;
+
+namespace TranslateV3Samples
+{
+    public static class TranslateV3ListGlossary
+    {
+        /// <summary>
+        /// List Glossaries
+        /// </summary>
+        public static void ListGlossariesSample(string projectId)
+        {
+            TranslationServiceClient translationServiceClient = TranslationServiceClient.Create();
+            ListGlossariesRequest request = new ListGlossariesRequest
+            {
+                ParentAsLocationName = new LocationName(projectId, "us-central1"),
+            };
+            PagedEnumerable<ListGlossariesResponse, Glossary> response = translationServiceClient.ListGlossaries(request);
+            // Iterate over pages (of server-defined size), performing one RPC per page
```

Comment:
Consider improving this comment, the code below is effectively iterating over a collection of Glossary not over pages. The response object is the one making an RPC call per page and "returning" each of the elements in that page individually.
Please take into account that a lot of people looking at these samples are looking at them precisely because they are not familiar with the libraries, and comments like these mike be very confusing.